### PR TITLE
Fast Food topic

### PR DESCRIPTION
**Pull request by Olliesama**

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4698,3 +4698,20 @@ label monika_omamori:
...
+    m "Even the vegetarian options can be awful."
+    m 1h "[player], do you have fast food often?"
+    m 1i "It's ok to have it every now and then..."
+    m "But please don't make it part of your daily lifestyle."
+    m 1e "If I were there I would make you something healthy instead..."
+    m 1m "Even though I can't cook very well yet."
+    m 1e"[player], try eating healthily, please? For me?"
+    m  "It would make me feel better to know you're looking after yourself."
+    m "Maybe once I come over we can learn to cook together!"
+    m 1j "Doesn't that sound fun, my love?"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Missing `return`.

**Comment by Olliesama** [comment]
Fixed

**Comment by FireyAmber** [comment]
> m 1h "[player], do you have fast food often?"

I feel like you could use a menu: here to expand the topic into several options here.

Other than that, the topic's good but it could use some expanding.

**Comment by Olliesama** [comment]
Updated, thanks for the help Amber.

**Comment by FaverosGema** [comment]
Since your topic alludes to having a healthier diet, and has a small reference to `monika_veggies`, (A topic where Monika confesses that she is vegetarian and suggests the `[player]` to try it out) there is an opportunity for expanding the connection to this topic.

You could even provide evidence for how fast food contributes to the carbon footprint resulting as an implication of cultivating livestock. Here is a .txt file for `monika_veggies`: [monika_veggies.txt](https://github.com/Backdash/MonikaModDev/files/1660435/monika_veggies.txt)

**Comment by Olliesama** [comment]
I don't intend to sound dismissive or rude but I really don't want to include any politics into a dialogue that was primarily just Monika being concerned about the player and his lifestyle.  >->
Is that ok? Sorry.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4698,3 +4698,44 @@ label monika_omamori:
     m 1k "I could be your guardian deity, ehehe~"
     return    
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel=monika_fastfood",category=['Monika'],prompt="Do you like fast food?",random=True))
```
Comment (monikLover) [inline Monika After Story/game/script-topics.rpy]: Missing an opening double quote on `label`

**Comment by Olliesama** [comment]
Fixed. :)

**Comment by Danymations** [comment]
@Olliesama Great topic! Here's edits that I made. Monika sounded sort of passive-aggressive here before, eheh~
```
label monika_fastfood:
    m 1m "I'm not a very big fan of it, I don't find it to taste very good"
    m  1d "Most places that either make the food or serve it usually put an unhealthy amount of chemicals in it."
    m 1n "Sometimes, even the vegetarian options are unhealthy."
    menu:
        m "[player], do you have fast food often?"

        "Yes, I do.":
            m 3d "It's okay to have it every once in a while, although I can't help but worry that you eat such bad food so often..."
            m 3g "When I finally get to see you in your reality, i'll try my best to cook healthier food for you. Even if I can't cook very well just yet."
            m 3n "Well, I know that just my love will be enough. I'll try my absolute best on cooking for you~"
            m 1q "..."
            m 1e "[Player], do you think you can try something out for me?"
            m 1e  "Do you think you can try and eat better? It just worries me that you're eating such unhealthy food so much."
            m 1q "It would break my heart to hear that you've become sick because of the food you buy."
            m 1h "I do understand that at times it's much easier to order out, since preparing your own food can be a hassle..."
            m 1e "But maybe you could see cooking as an opportunity to have fun!"
            m 1h "Or perhaps a skill for you to become really good at."
            m 1m "I would love to try your dishes, maybe you can do it for one of our dates, [player]~"
            m 1e "That would be really romantic, ehehe~"
            m "And that way we can both enjoy ourselves and you would be eating better."
            m 1a "Seems like a win-win to me, don't you think?"

        "No, I don't.":
            m 1r "That's great!"
            m 1a "I got sort of worried there for a second, [player]."
            m "I suppose that instead of eating out all the time, you cook your own food or someone else prepares it for you."
            m "Fast food can be really expensive over time, so doing it yourself is usually cheaper."
            m 1b "It also tastes a lot better a lot of the time!"
            m "Some people do find cooking overwhelming, and I understand that."
            m 1m "I believe it's because managing to keep enough ingredients is really inconvenient..."
            m 1m "And how long the food must take to prepare."
            m 1k "But I think I think the results are worth the effort."
            m 1e "Are you any good at cooking [player]?"
            m 1h "I believe you are. Whatever you cook, [player], I will gladly eat."
            m 5a "Well, as long as it's vegetarian. Ehehe~"

```

**Pull request merged**